packages/tracing/src/browser/browsertracing.ts

```diff
@@ -3,13 +3,25 @@ import { EventProcessor, Integration, Transaction as TransactionType, Transactio
 import { logger } from '@sentry/utils';
 
 import { startIdleTransaction } from '../hubextensions';
-import { DEFAULT_IDLE_TIMEOUT } from '../idletransaction';
+import { DEFAULT_IDLE_TIMEOUT, IdleTransaction } from '../idletransaction';
 import { Span } from '../span';
-
+import { SpanStatus } from '../spanstatus';
+
+import { registerBackgroundTabDetection } from './backgroundtab';
+import { registerErrorInstrumentation } from './errors';
+import { MetricsInstrumentation } from './metrics';
+import {
+  defaultRequestInstrumentionOptions,
+  registerRequestInstrumentation,
+  RequestInstrumentationOptions,
+} from './request';
 import { defaultBeforeNavigate, defaultRoutingInstrumentation } from './router';
+import { secToMs } from './utils';
+
+export const DEFAULT_MAX_TRANSACTION_DURATION__SECONDS = 600;
```

Comment:
```suggestion
export const DEFAULT_MAX_TRANSACTION_DURATION_SECONDS = 600;
```
This should probably be refactored in VSCode